osu.Game/Rulesets/Difficulty/DifficultyCalculator.cs

```diff
@@ -118,7 +117,7 @@ public List<TimedDifficultyAttributes> CalculateTimed([NotNull] IEnumerable<Mod>
                     skill.Process(hitObject);
                 }
 
-                attribs.Add(new TimedDifficultyAttributes(hitObject.EndTime * clockRate, CreateDifficultyAttributes(progressiveBeatmap, playableMods, skills, clockRate)));
+                attribs.Add(new TimedDifficultyAttributes(clock.GetTimeAt(hitObject.EndTime), CreateDifficultyAttributes(progressiveBeatmap, playableMods, skills, clock)));
```

Comment:
Isn't this doubly-applying the rate? `hitObject.EndTime` in this case is already clock-adjusted.